tls/s2n_security_policies.c

```diff
@@ -791,6 +799,7 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version = "CloudFront-TLS-1-0-2014", .security_policy = &security_policy_cloudfront_tls_1_0_2014, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "CloudFront-TLS-1-0-2016", .security_policy = &security_policy_cloudfront_tls_1_0_2016, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "CloudFront-TLS-1-1-2016", .security_policy = &security_policy_cloudfront_tls_1_1_2016, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "CloudFront-TLS-1-2-2017", .security_policy = &security_policy_cloudfront_tls_1_2_2017, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
```

Comment:
Can you explain the naming here? Why 2017? And should it be 1-3, since it contains 1.3 cipher suites, or is that also part of a date?

Reply:
The reasoning for 2017 was it's a copy of 2018 with a couple of additional ciphers from the 2016 policy, so it's somewhere between the two. Open to different naming suggestions. It should be 1.2. The cipher suite is the same as cipher_suites_cloudfront_tls_1_2_2018 with two additional ciphers added.

Reply:
Averaging dates seems like an odd versioning strategy. Why not 2022, since you're creating it in 2022?

And I'm guessing it's "1_2" because you're naming it using the minimum version supported, not the maximum version?

Reply:
> Averaging dates seems like an odd versioning strategy. Why not 2022, since you're creating it in 2022?

In our case, the higher year means the more secure set of ciphers we use, but this policy is worse then any of the 2018+ ciphers sets, so I'd rather not put a higher year on it.

> And I'm guessing it's "1_2" because you're naming it using the minimum version supported, not the maximum version?

Right